perf_dashboard/deploy/perf-dashboard/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+name: perf-dashboard
+description: A Helm chart for Istio perf-dashboard
+version: 0.1.0
+dependencies:
+- name: metrics-server
+  version: ^2.0.3
+  repository: https://kubernetes-charts.storage.googleapis.com/
+keywords:
+- istio
+- perf
+- perf-dashboard
+sources:
+- https://github.com/istio/tools
```

Comment:
Sources could be: https://github.com/istio/tools/perf_dashboard instead?

Reply:
Updated.